source/Generic/SplashScreen/SplashScreen.cs

```diff
@@ -331,48 +331,59 @@ private void SplashWindowClosed(object sender, EventArgs e)
 
         private string GetSplashVideoPath(Game game)
         {
-            var videoPathTemplate = Path.Combine(PlayniteApi.Paths.ConfigurationPath, "ExtraMetadata", "{0}", "{1}", "VideoIntro.mp4");
+            const string videoIntroName = "VideoIntro.mp4";
```

Comment:
Keep using `var` for consistency or make it a constant property outside of the method, whichever is fine

Reply:
Moved to other constants